src/wallet/crypter.h

```diff
@@ -162,28 +162,26 @@ class CCryptoKeyStore : public CBasicKeyStore
     {
         {
```

Comment:
Could this scope level be removed alongside with the `return false` in the end, or am I missing something about how `LOCK` works?